docs/FLYSKY_FRM303.md

```diff
@@ -101,4 +101,49 @@ Subsequent flashing can be done in two ways:
     - This enables programming over the USB-C port using the DFU method
     - Launch STM32CubeProgrammer and select USB as the connection method, click connect 
     - Proceed as usual
-   
+
+
+## Optional modifications ##
+
+### OLED display addition ###
+
+The transmitter side FRM303 can be optionally equipped with a 0.96" 128x64 pixel monochrome OLED screen with I2C bus and +3.3V supply. Current parameters can be observed and the settings can be of the TX as well as connected RX side mLRS module can be altered via 5-way joystick and OLED screen.
+
+<img src="images/FRM303-OLED-mod_finished.jpg" width="400px">
+
+To activate OLED screen on FRM303, a custom firmware build is required that defines `MLRS_FEATURE_OLED`.
+
+You can find two STL files to 3D print a mount and a housing for the OLED display under `docs/files/FRM303*.stl`
+
+<img src="images/FRM303-OLED-mod_parts.jpg" width="400px">
+
+The following pictures illustrate the assembly process in a couple of steps (click on the picture to open a bigger version):
+
+<a href="images/FRM303-OLED-mod_step1.jpg"><img src="images/FRM303-OLED-mod_step1.jpg" height="100px"></a> 
+<a href="images/FRM303-OLED-mod_step2.jpg"><img src="images/FRM303-OLED-mod_step2.jpg" height="100px"></a> 
+<a href="images/FRM303-OLED-mod_step3.jpg"><img src="images/FRM303-OLED-mod_step3.jpg" height="100px"></a> 
+<a href="images/FRM303-OLED-mod_step4.jpg"><img src="images/FRM303-OLED-mod_step4.jpg" height="100px"></a> 
+<a href="images/FRM303-OLED-mod_step5.jpg"><img src="images/FRM303-OLED-mod_step5.jpg" height="100px"></a> 
+<a href="images/FRM303-OLED-mod_step6.jpg"><img src="images/FRM303-OLED-mod_step6.jpg" height="100px"></a> 
+<a href="images/FRM303-OLED-mod_step7.jpg"><img src="images/FRM303-OLED-mod_step7.jpg" height="100px"></a>
+
+Please do note that a std. 0.96" OLED mounting holes need to be slightly enlarged to match the M2.5 screw holed of the FRM303 case.
+
+### mLRS-WiFi-bridge addition ###
+
+With mLRS-WiFi-bridge mod, you will be able to stream serial data (e.g. MAVLink) wirelessly from the FRM303 to a WiFi capable device, such as a computer, a tablet or a smartphone and use software such as [Mission Planner](https://ardupilot.org/planner/) to evaluate telemetry data or adjust flight controller parameters on the fly.
+
+Flysky FRM303 micro-JR bay adapter is spacious enough to withhold [M5Stack ATOM Lite ESP32 module](https://shop.m5stack.com/products/atom-lite-esp32-development-kit), together with [Adafruit MPM3610 5V voltage regulator](https://www.adafruit.com/product/4739). The result is a tidy and compact solution:
```

Comment:
Small note, just 'hold' instead of 'withhold'. "...spacious enough to hold [M5Stack..."

Reply:
Thx, fixed.